provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerTransformer.java

```diff
@@ -1,3 +1,130 @@
 package gov.va.api.health.providerdirectory.service.controller.practitioner;
 
-public class PractitionerTransformer {}
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.allBlank;
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.convert;
+
+import gov.va.api.health.providerdirectory.api.datatypes.Address;
+import gov.va.api.health.providerdirectory.api.datatypes.ContactPoint;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner.Gender;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner.PractitionerHumanName;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner.PractitionerIdentifier;
+import gov.va.api.health.providerdirectory.service.ProviderContacts;
+import gov.va.api.health.providerdirectory.service.ProviderResponse;
+import gov.va.api.health.providerdirectory.service.ProviderWrapper;
+import gov.va.api.health.providerdirectory.service.controller.EnumSearcher;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PractitionerTransformer implements PractitionerController.Transformer {
+
+  Boolean active(String active) {
+    if (active.toLowerCase().equals("active")) {
+      return true;
+    }
+    return false;
+  }
+
+  Address address(ProviderResponse.Value value) {
+    return convert(
+        value,
+        ppms ->
+            Address.builder()
+                .city(ppms.addressCity())
+                .country(ppms.addressCountry())
+                .state(ppms.addressStateProvince())
+                .build());
+  }
+
+  List<Address> addresses(ProviderResponse.Value value) {
+    if (value.address() == null) {
+      return null;
+    }
+    List<Address> addressList = new ArrayList<>();
+    addressList.add(address(value));
+    return addressList;
+  }
+
+  @Override
+  public Practitioner apply(ProviderWrapper ppmsData) {
+    return practitioner(ppmsData);
+  }
+
+  Practitioner.Gender gender(String gender) {
+    return convert(gender.toLowerCase(), ppms -> EnumSearcher.of(Gender.class).find(ppms));
```

Comment:
Potential null pointer here. The `convert` method can handle it if you pass `gender` in to `convert` directly:

```
return convert(gender, ppms -> EnumSearcher.of(Gender.class).find(ppms.trim().toLowerCase()));
```